.github/workflows/legacy-release_sbom-generator.yaml

```diff
@@ -33,41 +35,64 @@ jobs:
             # Use the input provided in manual run
             latest_tag=${{ github.event.inputs.dotcms_version }}
           fi
+          # Format the tag name: remove 'v' prefix
+          formatted_tag=$(echo "$latest_tag" | sed -e 's/^v//' -e 's/^dotcms-cli-//')
+          # Construct the branch name based on the formatted version
+          branch_name="release-${formatted_tag}"
+          echo "DOTCMS_VERSION=${formatted_tag}" >> $GITHUB_ENV
+          echo "BRANCH_NAME=${branch_name}" >> $GITHUB_ENV
```

Comment:
For consideration: 
consider making this a standalone `set dotCMS release version` step.  Or adjust the name & id of the existing combined step to be clear that it's a get&set step.